### DISCUSSION TITLE
[A-L] (3 Aprile 2023) Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#382] [Esami A-L]
Siano $\Large R_1$ e $\Large R_2$ due relazioni binarie su un insieme $\Large A$.

Indicare se le seguenti affermazioni sono vere o false.

1. Se $\Large R_1 \cup R_2$ non è simmetrica allora $\Large R_1$ non è simmetrica o $\Large R_2$ non è simmetrica.

2. Se $\Large R_1$ e $\Large R_2$ sono transitive allora $\Large R_1 \cup R_2$ è transitiva.

3. Se $\Large R_1 \cap R_2$ non è riflessiva e simmetrica allora $\Large R_1 \cup R_2$ non è riflessiva.

**Comment by cek753** [comment]
1 V
2 F
3 F
